.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,11 @@
-to: @airbnb/streamalert-maintainers
```

Comment:
I would prefer to keep `@airbnb/streamalert-maintainers` either in `to` or `cc` field. Otherwise, it is difficult to find out who to tag to.

Reply:
sorry I didn't mean to remove that - adding back

Reply:
but adding to the cc line instead of to